docs/UserGuide.md

```diff
@@ -90,73 +91,79 @@ interactions recorded faster and simpler than traditional GUI applications.
 
 ### Adding a client: `add`
 
-Adds an empty client to the list without any Remark and transactions.
+Adds a client to the list.
 
-Format: `add n/NAME a/ADDRESS p/PHONE e/EMAIL [t/TAG]`
+Format: `add n/NAME a/ADDRESS p/PHONE e/EMAIL [t/TAG]...`
 
 * Tags are optional.
 * Multiple tags can be tagged to the client.
-* Use `create` command to add Remark to the client.
 
 Examples:
-* `add n/MacDonalds a/West Coast Park p/9876542 e/mcd@mcd.com`
-* `add n/KFC a/Yishun Street 81 p/9876543 e/kfc@kfc.com t/FriedChicken`
+* `add n/Alice a/West Coast Park p/9876542 e/alice@gmail.com`
+* `add n/John a/Yishun Street 81 p/9876543 e/john@yahoo.com t/friends t/supplier`
 
 ### Clearing all entries : `clear`
 
 Clears all entries from JeeqTracker.
```

Comment:
Maybe, it might be better to specify what does `all entries` refer to?

Reply:
Sure, will update it